core/domain/draft_upgrade_services.py

```
@@ -312,6 +312,112 @@ def _convert_html_in_draft_change_list(
                 })
         return draft_change_list
 
+    @classmethod
+    def _convert_states_v52_dict_to_v53_dict(
+        cls, draft_change_list: List[exp_domain.ExplorationChange]
+    ) -> List[exp_domain.ExplorationChange]:
+        """Converts from version 52 to 53. Version 53 fixes general
+        state, interaction and rte data. This will update the drafts
+        for state and RTE part but won't be able to do for interaction.
+        The `ExplorationChange` object for interaction is divided into
+        further properties and we won't be able to collect enough info
+        to update the draft.
```

For this part should we just throw the draft away, by raising `InvalidDraftConversionException`

---

I think we can't do that, so mainly the value that I validate or correct inside interaction is of answer groups and I do have some basic checks for the answer group as well which belongs to the `general state checks`. So I think we can't do that.

---

Sorry, not sure what you mean. This is just about drafts, these can always be removed.